src/main/java/org/ambraproject/rhino/service/taxonomy/impl/TaxonomyClassificationServiceImpl.java

```diff
@@ -297,7 +294,8 @@ public void populateCategories(ArticleRevision revision) {
         .collect(Collectors.toList());
   }
 
-  private void persistCategories(List<WeightedTerm> terms, Article article) {
+  @VisibleForTesting
```

Comment:
public and visibleForTesting?

Reply:
Yeah, that means this is basically a `private` method but we had to make it public so that it could be tested. I could probably use the default visibility instead though.

Reply:
I don't think you need the annotation if it's public

Reply:
https://guava.dev/releases/19.0/api/docs/com/google/common/annotations/VisibleForTesting.html